core/test/financial_templates/ExpiringMultiPartyCreator.js

```diff
@@ -0,0 +1,93 @@
+const { toWei } = web3.utils;
+
+const truffleAssert = require("truffle-assertions");
+const { RegistryRolesEnum } = require("../../../common/Enums.js");
+
+// Tested Contract
+const ExpiringMultiPartyCreator = artifacts.require("ExpiringMultiPartyCreator");
+
+// Helper Contracts
+const Finder = artifacts.require("Finder");
+const ERC20MintableData = require("@openzeppelin/contracts/build/contracts/ERC20Mintable.json");
```

Comment:
replace this with `artifacts.require("Token")` which is our own ERC20 and removes dependence on openzeppelinv2.5 which we are moving away from. See #955